packages/experiments/src/components/PersonaCoin/examples/PersonaCoin.Example.tsx

```diff
@@ -29,6 +29,7 @@ export class PersonaCoinExample extends React.Component<{}, {}> {
           <Stack tokens={tokens.personaCoinStack}>
             <Text>When passing text initials will be extracted from the text</Text>
             <PersonaCoinStack>
+              <PersonaCoin text="Kevin Jameson" />
```

Comment:
Nit, did you mean to dupe this?

Reply:
This is one of some temporary examples I forgot to remove... will do.